boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -187,19 +218,46 @@ openapi-generate:
 			-p % \
 			-h /dev/null \
 			-r "-"
+else
+	find ./api -maxdepth 2 -mindepth 1 -type d | xargs -t -n1 -I% \
+		openapi-gen --logtostderr=true \
+			-i % \
+			-o "" \
+			-O zz_generated.openapi \
+			-p % \
+			-h /dev/null \
+			-r "-"
+endif
```

Comment:
At a glance, these look to be the same except for the directory (already has a variable, as noted above) and the mindepth. If you added a variable for the latter, you could dedup this.